src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -141,9 +141,9 @@ public void Write(JsonTextWriter writer)
 
             this.EmitParametersIfPresent(jsonWriter, emitter);
 
-            jsonWriter.WritePropertyName("functions");
-            jsonWriter.WriteStartArray();
-            jsonWriter.WriteEndArray();
+            // jsonWriter.WritePropertyName("functions");
```

Comment:
You should remove these lines rather than comment them out. We can always recover these from history if they're ever needed again. In general, commenting out blocks or statement is ok for temporary changes, but otherwise we try to remove code that isn't needed anymore.

Reply:
Thanks! I changed that in the new commit.